### ISSUE TITLE
bowtie2 alignment mode

### ISSUE THREAD

**Issue by donutbrew** [LANL-Bioinformatics/EDGE#25]
I see that for normal reads (not long reads), bowtie2 is set to run in global mode instead of a local mode. This can cause issues in generating consensus sequences in some poorly-trimmed fastq files. I would suggest either to change to use '--local or' '--sensitive-loca'l or to label the bowtie2 button as global alignment (as this may be what you're going for).

**Comment by chienchi** [comment]
Yes.  We use default mode for the aligners.  If users would like to turn on specific mode, users can use "Aligner Options" to change the parameters as you suggest here with --local or --sensitive-local. Or, use BWA mem as aligner which will do local alignemnt. Another example is to use "-x ont2d" to run BWA mem with Nanopore reads. And I agree with you that we should put descriptions about the aligners in the documentation site.
